packages/console/src/components/CreateConnectorForm/index.tsx

```diff
@@ -52,6 +52,8 @@ function CreateConnectorForm({ onClose, isOpen: isFormOpen, type }: Props) {
         // Hide the entrance of adding SAML social connectors, users should go to Enterprise SSO if they want to use SAML.
         // Should not remove the SAML factory from GET /connector-factories API, since that could break the existing SAML connectors.
         .filter(({ id }) => id !== 'saml')
+        // Hide the entrance of adding HTTP Email connector
+        .filter(({ id }) => id !== 'http-email')
```

Comment:
can we add a hidden flag or sth?

Reply:
maybe later, until we have more connectors.